src/fides/core/deploy.py

```diff
@@ -89,9 +90,17 @@ def check_docker_version(
             raise DockerCheckException(
                 "Could not determine Docker version from 'docker' commands. Please ensure that Docker is installed and running and try again."
             )
-        parsed_version = (
+        stripped_version = (
             raw_version.stdout.decode("utf-8").rstrip("\n").replace("'", "")
         )
+        parsed_version = re.match(r"((\d+\.)+\d+)", stripped_version)
```

Comment:
used `match` instead of `search` because i think we can assume the version number is the beginning of the string, especially after stripping it above???

Reply:
Eh... if there's just a single preceding space ` ` you'll be left with nothing!

The `rstrip("\n")` seems to just strip trailing newlines, not touch whitespace at all.

TL;DR: I would chain one more `.strip()` to the parsing

Reply:
ooh, right didn't pay close enough attention to what we were stripping. i'll do as you suggest, that feels reasonable.